.github/workflows/create-release.yml

```diff
@@ -156,7 +156,23 @@ jobs:
 
     - uses: ncipollo/release-action@v1
       name: Create release
+      id: create_release
       with:
         tag: ${{ github.event.inputs.milestone_title }}
         token: ${{ secrets.GITHUB_TOKEN }}
         bodyFile: ".github/workflows/release-body.md"
+
+    - name: Create ${{ github.event.inputs.milestone_title }} Native Windows install script
+      run: |
+        cp install_sct.bat install_sct-${{ github.event.inputs.milestone_title }}_win.bat
+        perl -pi -e 's/set git_ref=master/set git_ref=${{ github.event.inputs.milestone_title }}/' install_sct-${{ github.event.inputs.milestone_title }}_win.bat
+    
+    - uses: xresloader/upload-to-github-release@v1
```

Comment:
It looks like `ncipollo/release-action@v1` supports an `artifacts` input parameter (list [here](https://github.com/ncipollo/release-action#action-inputs)), would that also work? Assuming the previous step (Create Native Windows install script) is reordered to happen before the "Create release" step.

Reply:
Suggestion tested in https://github.com/joshuacwnewton/spinalcordtoolbox/commit/48e0df56cb9665cf9a26d828944b1c9cb37981ed and https://github.com/joshuacwnewton/spinalcordtoolbox/releases/tag/5.7.6

LGTM! Thank you for this. :)